.vscode/extensions.json

```diff
@@ -2,8 +2,9 @@
     // See http://go.microsoft.com/fwlink/?LinkId=827846
     // for the documentation about the extensions.json format
     "recommendations": [
-        "ms-vscode.cpptools",
-        "platformio.platformio-ide",
-        "trunk.io"
+        "platformio.platformio-ide"
     ],
-}
\ No newline at end of file
+    "unwantedRecommendations": [
+        "ms-vscode.cpptools-extension-pack"
+    ]
+}
```

Comment:
Please dont change the extenstions file...

Reply:
Get corrected thx

Reply:
> Please dont change the extenstions file...

Can we add this to .gitignore?

Reply:
I have mixed feelings and that. It is nice to have it recommend pio and trunk for folks cloning the repo for the first time, but on the other hand, this extension pack bit is real pain. It gets me all of the time

Reply:
You can set `git update-index --assume-unchanged .vscode/extensions.json` in your repo, then this file will never show up as modified even if you change it. It's a local setting only, though.